docs/learn/beginner/03-accounts.md

```diff
@@ -21,44 +21,25 @@ In the Cosmos SDK, an _account_ designates a pair of _public key_ `PubKey` and _
-|    Address 0     |              |    Address 1     |               |    Address 2     |
-|        ^         |              |        ^         |               |        ^         |
-|        |         |              |        |         |               |        |         |
-|        |         |              |        |         |               |        |         |
-|        |         |              |        |         |               |        |         |
-|        +         |              |        +         |               |        +         |
-|  Public key 0    |              |  Public key 1    |               |  Public key 2    |
-|        ^         |              |        ^         |               |        ^         |
-|        |         |              |        |         |               |        |         |
-|        |         |              |        |         |               |        |         |
-|        |         |              |        |         |               |        |         |
-|        +         |              |        +         |               |        +         |
-|  Private key 0   |              |  Private key 1   |               |  Private key 2   |
-|        ^         |              |        ^         |               |        ^         |
-+------------------+              +------------------+               +------------------+
-         |                                 |                                  |
-         |                                 |                                  |
-         |                                 |                                  |
-         +--------------------------------------------------------------------+
-                                           |
-                                           |
-                                 +---------+---------+
-                                 |                   |
-                                 |  Master PrivKey   |
-                                 |                   |
-                                 +-------------------+
-                                           |
-                                           |
-                                 +---------+---------+
-                                 |                   |
-                                 |  Mnemonic (Seed)  |
-                                 |                   |
-                                 +-------------------+
+```mermaid
+graph TD
+    A0[Account 0] --> A0A[Address 0]
+    A0A --> A0PK[Public key 0]
+    A0PK --> A0SK[Private key 0]
+
+    A1[Account 1] --> A1A[Address 1]
+    A1A --> A1PK[Public key 1]
+    A1PK --> A1SK[Private key 1]
+
+    A2[Account 2] --> A2A[Address 2]
+    A2A --> A2PK[Public key 2]
+    A2PK --> A2SK[Private key 2]
+
+    A0SK --> MasterPK[Master PrivKey]
+    A1SK --> MasterPK[Master PrivKey]
+    A2SK --> MasterPK[Master PrivKey]
+
+    MasterPK --> Mnemonic["Mnemonic (Seed)"]
```

Comment:
The direction of the arrows are different from the original, chat gpt helped me out fixing it :D

```suggestion
graph BT
    A0A[Address 0] --> A0[Account 0]
    A0PK[Public key 0] --> A0A[Address 0]
    A0SK[Private key 0] --> A0PK[Public key 0]
    A1A[Address 1] --> A1[Account 1]
    A1PK[Public key 1] --> A1A[Address 1]
    A1SK[Private key 1] --> A1PK[Public key 1]
    A2A[Address 2] --> A2[Account 2]
    A2PK[Public key 2] --> A2A[Address 2]
    A2SK[Private key 2] --> A2PK[Public key 2]
    MasterPK[Master PrivKey] --> A0SK[Private key 0]
    MasterPK[Master PrivKey] --> A1SK[Private key 1]
    MasterPK[Master PrivKey] --> A2SK[Private key 2]
    Mnemonic["Mnemonic (Seed)"] --> MasterPK[Master PrivKey]
```